docs/platforms/index.mdx

```diff
@@ -20,13 +20,18 @@ We also <Link to="https://sentry.zendesk.com/hc/en-us/articles/23961784367387">d
 
 </Note>
 
-## Platforms
-
-<PlatformGrid />
+<PlatformFilter />
 
 ## Community Supported
 
-These SDKs are maintained and supported by [the Sentry community](https://open.sentry.io/). While generally our community does a great job at responding to issues, it's important to understand that Sentry staff cannot help you with issues using a community-supported SDK.
+<div style={{display: 'flex', flexWrap: 'wrap', gap: '0.5rem'}}>
+  <div style={{width: '100%', maxWidth: '50ch'}}>
+    These SDKs are maintained and supported by [the Sentry community](https://open.sentry.io/). While generally our community does a great job at responding to issues, it's important to understand that Sentry staff cannot help you with issues using a community-supported SDK.
+  </div>
+  <div style={{diplay: 'inline', flex: 1, minWidth: '120px', maxWidth: '250px'}}>
+    ![Community Supported](./community.png)
+  </div>
```

Comment:
```suggestion
  <div style={{diplay: 'inline', flex: 1, minWidth: '120px', maxWidth: '250px'}}>
    ![Community Supported](./community.png)
  </div>
```
Is there a way to get rid of the drop shadow around the image?
Also, what do you think about having the image here instead of on the side?
<img width="275" alt="Screenshot 2024-11-26 at 1 48 17 PM" src="https://github.com/user-attachments/assets/5cdbfb10-6cde-4999-a77e-9f0d83adea7d">